src/main/java/command/Command.java

```diff
@@ -0,0 +1,22 @@
+package command;
+
+import exception.DukeException;
+import logic.Storage;
+import logic.Ui;
+import tasks.TaskList;
+
+/**
+ * Abstract class of all the commands that is created
+ * after parsing the user's input
+ */
+
```

Comment:
Should this empty line be deleted? I don't think it's necessary to have an empty line separating the javadocs and the class